test/behavior/error.zig

```diff
@@ -208,8 +208,29 @@ fn testErrorSetType() !void {
     }
 }
 
+test "error set cast at comptime" {
```

Comment:
no need for this extra test. `comptime try testExplicitErrorSetCast(Set1.A); ` below is already doing a comptime version of the same test.

Reply:
Yeah I thought so too, but it wasn't because of the `var` usage? Maybe that's a bug in stage2. I'll remove this test for now.